Modules/Core/Common/include/itkSymmetricEigenAnalysis.h

```diff
@@ -412,7 +412,7 @@ class ITK_TEMPLATE_EXPORT SymmetricEigenAnalysis
    *  num. math. 11, 181-195(1968) by martin, reinsch, and wilkinson.
    *    handbook for auto. comp., vol.ii-linear algebra, 212-226(1971).    */
   void
-  ReduceToTridiagonalMatrixAndGetTransformation(double * a, double * d, double * e, double * z) const;
+  ReduceToTridiagonalMatrixAndGetTransformation(const double * a, double * d, double * e, double * z) const;
```

Comment:
This PR includes a commit from the other PR.

Reply:
Yes. There is a dependency.

Hans